specification/batch/data-plane/Microsoft.Batch/stable/2022-01-01.15.0/BatchService.json

```diff
@@ -14496,7 +14527,7 @@
           "type": "integer",
           "format": "int32",
           "title": "The number of low-priority Compute Nodes currently in the Pool.",
-          "description": "low-priority Compute Nodes which have been preempted are included in this count."
+          "description": "Low-priority Compute Nodes which have been preempted are included in this count."
```

Comment:
Let's do a find/replace on Low-priority and change to "Spot/low-priority" (with Spot capitalized and low-priority lowercase to match other docs)